testdata/well_known/make_well_known_custom_order.py

```diff
@@ -0,0 +1,245 @@
+"""
+This script is designed to convert a SEGY file into a VDS file format.
+One of the features that it offers is the ability to customize the order of the 
+"xline," "inline," and "sample" values to suit your specific needs. 
+
+To change the order, set the desired order at line [60] in "axisDescriptor" 
+variable and run the script.
+
+Usage:
+    python make_well_known_custom_order.py [--segyfile][--vdsfile][--crsWkt]
+
+Options:
+    --segyfile [Requird]
+    --vdsfile  [default: "well_known_custom_order.vds"]
+    --crsWkt   [default: "utmXX"]
```

Comment:
I also have no clue how important that parameter is. I made this as a script arg because we are passing this as an argument to the script that creates the SEGY file. I think we can keep this as an argument for now and set the default value.